src/osemgrep/reporting/Output.ml

```diff
@@ -10,15 +11,40 @@ module OutJ = Semgrep_output_v1_j
 
    Partially translated from output.py
 
-   LATER? It would be nice to move this file in osemgrep/reporting/, but
-   it currently depends on Scan_CLI.conf and Core_runner so simpler to keep
-   here for now.
-
-   We're using Out.put() below, not Logs.app(), because we want to output
+   We're using Common.pr() below, not Logs.app(), because we want to output
    findings on stdout (Logs.app uses stderr). That also mean semgrep will
    display findings even with --quiet.
 *)
 
+type conf = {
+  nosem : bool;
+  autofix : bool;
+  dryrun : bool;
+  strict : bool;
+  (* maybe should define an Output_option.t, or add a record to
+   * Output_format.Text *)
+  force_color : bool;
+  logging_level : Logs.level option;
```

Comment:
It's actually in both. It's in the common config and here, because I wasn't sure how else to abstract it. The alternative is to have the logging level be an explicit parameter of the Output.* functions

Reply:
Could we perhaps remove `logging_level` and instead rely on `log_config.json` for configuration?

e.g.
```
{
  "loggers": [
    {
      "name": "Main.Core_scan",
      "level": "info",
      "handlers": [
        {
          "cli": {
            "level": "info"
          }
        }
      ]
    },
    {
      "name": "Main.Pfff_or_tree_sitter",
      "level": "trace",
      "handlers": [
        {
          "cli": {
            "level": "trace"
          }
        }
      ]
    }
  ]
}
```